internals/overlord/state/identities.go

```diff
@@ -102,15 +152,22 @@ func (d *Identity) UnmarshalJSON(data []byte) error {
 	identity := Identity{
 		Access: IdentityAccess(ai.Access),
 	}
-	switch {
-	case ai.Local != nil:
+
+	if ai.Local != nil {
 		if ai.Local.UserID == nil {
 			return errors.New("local identity must specify user-id")
 		}
 		identity.Local = &LocalIdentity{UserID: *ai.Local.UserID}
 	}
+	if ai.Basic != nil {
+		if ai.Basic.Password == "" {
+			return errors.New("basic identity must specify password (hashed)")
+		}
```

Comment:
I think this is unnecessary because is checked in `validateExcludingName`, which is called below, right?